code/modules/power/solar.dm

```diff
@@ -191,7 +191,7 @@
 	var/glass_type = null
 
 /obj/item/solar_assembly/attack_hand(mob/user)
-	if(!anchored && isturf(loc)) // You can't pick it up
+	if(in_storage || (!anchored && isturf(loc))) // You can't pick it up
```

Comment:
```suggestion
	if(!anchored)
```
I don't see much reason for the `isturf(loc)` check myself. Without this the code will function just fine as expected. Being able to pick it up, switching places in your hands and of course removing it from your backpack

Reply:
Not sure if the `isturf(loc)` is needed but I'll add this